main.go

```diff
@@ -782,9 +795,20 @@ func postCollaboratorPageHandler(c *gin.Context, db *sql.DB) {
 	}
 
 	repoId := c.PostForm("repo_id")
-	userId := c.PostForm("user_id")
 
-	_, err = db.Exec(`INSERT INTO collaborators(repo_id,user_id)VALUES($1,$2,)`, repoId, userId)
+	userName := c.PostForm("user_name")
+
+	var userId string
+
+	row := db.QueryRow(`SELECT id from users WHERE username = $1`, userName)
```

Comment:
Instead of custom query use GetUserByUsername function we already wrote.

Reply:
DONE